src/java/org/apache/cassandra/cql3/statements/TransactionStatement.java

```diff
@@ -380,82 +376,73 @@ public ResultMessage execute(QueryState state, QueryOptions options, Dispatcher.
     {
         checkTrue(DatabaseDescriptor.getAccordTransactionsEnabled(), TRANSACTIONS_DISABLED_MESSAGE);
 
-        try
```

Comment:
removed the try as it was just logging timeout/preemept/etc... we now have the coordinator / accord metrics that show the same thing so no longer really need the logger